.github/workflows/CI.yml

```diff
@@ -68,3 +68,14 @@ jobs:
         run: sudo apt install -y dotnet6
       - name: Run script to make sure all files end with EOL 
         run: dotnet fsi eofConvention.fsx
+      - name: Install prettier
+        run: npm install prettier@2.8.3
+      - name: Change file permissions
+        # We need this step so we can change the files using `npx prettier --write` in the next step.
+        # Otherwise we get permission denied error in the CI.
+        run: sudo chmod 777 -R .
+      - name: Run "prettier" to check the style of our TypeScript code
+        run: |
+          sudo npx prettier --quote-props=consistent --write './**/*.ts'
+          git config core.fileMode false
```

Comment:
@realmarv I suggested this workaround so that you dont need to do weird things with chmod; either use this or chmod but not both

Reply:
I thought this command makes git ignore mode changes in files and doesn't include it in git diff command. How does it solve permission denied error?

Reply:
oh, I thought the chmod thing was solving git diff showing mask file differences, my bad; anyway, then again you need to explain with a comment what that line is doing because it's not self-explanatory